scss/_patterns_section-and-block.scss

```diff
@@ -0,0 +1,17 @@
+@mixin vf-container-spacing-vertical($spacing) {
+  margin-bottom: calc($spacing / 2) !important;
+
+  @media (min-width: $breakpoint-large) {
+    margin-bottom: calc($spacing) !important;
+  }
```

Comment:
What is the `!important` for?

Reply:
I think it was a leftover from utilities that were used before, should be removed.